scripts/cstyle.pl

```diff
@@ -829,8 +877,8 @@ ($$)
 		$cont_first = 0;
 
 		# Strings may be pulled back to an earlier (half-)tabstop
-		unless ($cont_noerr || /^$cont_base    / ||
-		    (/^\t*(?:    )?(?:gettext\()?\"/ && !/^$cont_base\t/)) {
+		unless ($cont_noerr || /^$cont_base    / || /;/ ||
+		    (/^\t*(?:    )?(?:gettext\()?\"/ && !/^$cont_base\t/ && !/;/)) {
```

Comment:
Why was this hunk added?  We should remove it and update the code as needed.  Unless you have a specific example in the code base?